index.bs

```diff
@@ -1359,6 +1359,13 @@ whether to use an attribute or a method.
 - Attribute getters must not have any (observable) side effects.
     If you have expected side effects, use a method.
 
+- Attribute getters should not throw exceptions.
+    Exceptions are side effects.
+    Updating existing APIs to include getters that throw exception must be avoided.
```

Comment:
I think it can still be reasonable if that only happens in a new state that was not accessible before the update, but this is an interesting observation.

Reply:
Given @annevk's comment, perhaps must should be watered down a bit?